go/pkg/cosmos/types.go

```diff
@@ -96,7 +96,8 @@ type Tx struct {
 }
 
 type TxHistory struct {
-	TotalPages int
+	TotalPages int // Deprecated
```

Comment:
We can just remove since tx history endpoint has never been used/fully completed

Reply:
👍